### PR TITLE
Fixes hints for temperature concept

### PR DESCRIPTION
**Pull request by thepabloaguilar**
Hi, I was doing the `Temperature` exercise when I took a look on `HINTS.md` and felt like the formulas for each conversion was maybe swapped! This PR fixes that.

cc @pwadsworth

### CODE REVIEW

**Comment by petertseng** [comment]
you're right!

File: exercises/concept/temperature/.docs/hints.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Hints
 
-## Convert Farenheit to Celsius
+## Convert Fahrenheit to Celsius
```
Comment (ErikSchierboom) [inline exercises/concept/temperature/.docs/hints.md:3]: While we're at it, we should also correctly link the hint to the task:

```suggestion
## 1. Convert Fahrenheit to Celsius
```

File: exercises/concept/temperature/.docs/hints.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-- Temperature in degrees Celsius is defined as: T(°F) = T(°C) × 1.8 + 32
-- The temperature in Farenheit is provided as an `Integer` and must return Celsius as a `Float`.
+- Temperature in degrees Celsius is defined as: T(°C) = (T(°F) - 32) / 1.8
+- The temperature in Fahrenheit is provided as an `Integer` and must return Celsius as a `Float`.
   The `fromInteger` function will come in handy.
 
-## Convert Celsius to Farenheit
+## Convert Celsius to Fahrenheit
```
Comment (ErikSchierboom) [inline exercises/concept/temperature/.docs/hints.md:9]: While we're at it, we should also correctly link the hint to the task:

```suggestion
## 2. Convert Celsius to Fahrenheit
```

**Review by ErikSchierboom — APPROVED** [review]
Lovely!

**Pull request merged**